apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.html.ts

```diff
@@ -158,6 +158,107 @@ export function InstructorAssessmentSettings({
                 belongs to.
               </small>
             </div>
+            <div class="mb-3">
+              <label for="text">Text</label>
+              <textarea class="form-control" id="text" name="text" ${canEdit ? '' : 'disabled'}>
+${resLocals.assessment.text}</textarea
+              >
+              <small class="form-text text-muted">
+                HTML text shown on the assessment overview page.
+              </small>
+            </div>
+            <div class="mb-3">
+              <div class="mb-3 form-check">
+                <input
+                  class="form-check-input"
+                  type="checkbox"
+                  id="allow_issue_reporting"
+                  name="allow_issue_reporting"
+                  ${canEdit ? '' : 'disabled'}
+                  ${resLocals.assessment.allow_issue_reporting ? 'checked' : ''}
+                />
+                <label class="form-check-label" for="allow_issue_reporting">
+                  Allow issue reporting
+                </label>
+                <div class="small text-muted">
+                  Whether to allow students to report issues for assessment questions.
+                </div>
+              </div>
+            </div>
+            ${resLocals.assessment.type !== 'Homework'
+              ? html`
+                  <div class="mb-3 form-check">
+                    <input
+                      class="form-check-input"
+                      type="checkbox"
+                      id="multiple_instance"
+                      name="multiple_instance"
+                      ${canEdit ? '' : 'disabled'}
+                      ${resLocals.assessment.multiple_instance ? 'checked' : ''}
+                    />
+                    <label class="form-check-label" for="multiple_instance">
+                      Multiple Instances
```

Comment:
For consistency with other labels (none of the other ones use sentence case):

```suggestion
                      Multiple instances
```